src/app.ts

```diff
@@ -1,4 +1,5 @@
-import { make } from './utils';
+import { make } from './utils/make';
+import { sendDataToNotion } from './utils/notionHelpers';
 import classes from './styles/collapsed.module.css';
 
 /**
```

Comment:
I would suggest organizing an entry point as a class. And split `createWidget` for several single-task methods. Maybe it could be done later.